Pod/Classes/SuperMockResponseHelper.swift

```diff
@@ -86,12 +86,23 @@ class SuperMockResponseHelper: NSObject {
     }
     
     private func mockURLForRequestURL(url: NSURL, requestMethod: RequestMethod, mocks: Dictionary<String,AnyObject>) -> NSURL? {
-        
         guard let definitionsForMethod = mocks[requestMethod.rawValue] as? Dictionary<String,AnyObject> else {
-            fatalError("Couldn't find definitions for request: \(requestMethod) make sure to create a node for it in the plist")
+            if SuperMockConfig.sharedConfig.useStrictMocks {
+                fatalError("Couldn't find definitions for request: \(requestMethod) make sure to create a node for it in the plist")
+            } else {
+                return url
+            }
         }
-        
-        if let responseFile = definitionsForMethod[url.absoluteString] as? NSString,
+
+        let config = SuperMockConfig.sharedConfig
+        let trasformedURL: NSURL = config.URLTransform?(url:url) ?? url
+
+        if config.logURLTransforms && (trasformedURL != url) {
+            print("IncomingURL: \(url) Transormed: \(trasformedURL)")
+        }
+
+
+        if let responseFile = definitionsForMethod[trasformedURL.absoluteString] as? NSString,
```

Comment:
can this be conditionally casted to a String rather than NSString, the NSString method you use on the next line should bridge over.